examples/small-study/src/background.js

```diff
@@ -30,17 +30,16 @@
  *
  */
 
-class FeatureToInstrument {
+class StudyLifeCycleHandler {
```

Comment:
I am not keen on this, because it should about the Feature is the goal here, and the study code should be minor part.

Reply:
I claim we call it instrumented feature, and add a *tiny bit* more to it, such as a button or page action.

Reply:
I don't like the fact of merging boilerplate code (background.js) with study-specific implementations (https://github.com/mozilla/shield-studies-addon-template/blob/b27a17f3ca548b613150d229d471ce903a9355c7/src/Feature.js#L19-L77 is still a very small implementation)
Say we improve the best practice boilerplate code (background.js). It should then remain easy to update existing add-ons. If we merge these to concerns (listening/reacting to study life-cycle events + the actual implementation of the reactions) it will be harder to upgrade add-ons to newer versions of utils.

Reply:
My bid: Merge this without changing it, then create a new PR with a rename and we continue the discussion there? :)